src/main/java/Main.java

```diff
@@ -27,20 +24,26 @@ public static Session getSession() throws HibernateException {
         return ourSessionFactory.openSession();
     }
 
-    public static void main(final String[] args) throws Exception {
+    public static void main(final String[] args) {
         final Session session = getSession();
         try {
-            System.out.println("querying all the managed entities...");
+            log.info("querying all the managed entities...");
```

Comment:
Instead of using System.out.println, logging the messages you want to show is a better practice.